docker-compose.yml

```diff
@@ -186,22 +186,33 @@ services:
     ports:
       - "8964:8000"
 
+  dev-docs:
+    build:
+      context: .
+      target: docs
+    image: warehouse:docker-compose-docs
+    volumes:
+      - ./bin:/opt/warehouse/src/bin:z
+      - ./docs:/opt/warehouse/src/docs:z
+
   user-docs:
-    image: warehouse:docker-compose
+    image: warehouse:docker-compose-docs
+    pull_policy: never
     command: mkdocs serve -a 0.0.0.0:8000 -f mkdocs-user-docs.yml
     volumes:
+      - ./bin:/opt/warehouse/src/bin:z
       - ./mkdocs-user-docs.yml:/opt/warehouse/src/mkdocs-user-docs.yml:z
-      - ./docs/user:/opt/warehouse/src/docs/user:z
+      - ./docs:/opt/warehouse/src/docs:z
     ports:
       - "10000:8000"
-    depends_on: [web]
 
   blog:
-    image: warehouse:docker-compose
+    image: warehouse:docker-compose-docs
+    pull_policy: never
     command: mkdocs serve -a 0.0.0.0:8000 -f docs/blog.yml
     volumes:
-      - ./docs/blog.yml:/opt/warehouse/src/docs/blog.yml:z
-      - ./docs/blog:/opt/warehouse/src/docs/blog:z
+      - ./.git:/opt/warehouse/src/.git:z
```

Comment:
I had a similar note about git directory previously 
https://github.com/pypi/warehouse/pull/13231#discussion_r1142476044

If it needs to be mounted, should it be in ro mode?

Reply:
oh heck yeah, good call. it's just for generating the timestamps on blogs. i missed it getting removed.